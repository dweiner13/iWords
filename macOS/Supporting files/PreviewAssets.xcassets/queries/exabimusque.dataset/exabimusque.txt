que                  TACKON                             
-que = and (enclitic, translated before attached word); completes plerus/uter;
ex                   PREFIX                             
- out, away from; beyond; completely;
abi.mus              V      6 1 PRES ACTIVE  IND 1 P    
abeo, abire, abivi(ii), abitus  V INTRANS  
depart, go away; go off, go forth; pass away, die, disappear; be changed;

