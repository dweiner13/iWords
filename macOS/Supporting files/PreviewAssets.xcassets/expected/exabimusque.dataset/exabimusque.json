[{"definition":{"_0":{"words":[{"meaning":"-que = and (enclitic, translated before attached word); completes plerus\/uter;"}],"truncated":false,"possibilities":[{"tackon":{"_0":"que"}}]}}},{"definition":{"_0":{"words":[{"meaning":"- out, away from; beyond; completely;"}],"truncated":false,"possibilities":[{"prefix":{"_0":"ex"}}]}}},{"definition":{"_0":{"words":[{"meaning":"depart, go away; go off, go forth; pass away, die, disappear; be changed;","expansion":{"verb":{"_0":"abeo, abire, abivi(ii), abitus","_2":["intrans."]}}}],"truncated":false,"possibilities":[{"verb":{"_0":{"person":1,"number":"P","tense":"PRES","variety":1,"text":"abi.mus","voice":"ACTIVE","mood":"IND"}}}]}}}]